cmd/cinder-csi-plugin/main.go

```diff
@@ -100,6 +109,20 @@ func main() {
 
 func handle() {
 
+	if httpEndpoint != "" {
+		metricsManager := metrics.NewCSIMetricsManager("" /* driverName */)
```

Comment:
@jichenjc why cant we add drivr name directly in L113 , Is it necessary to set in L117